server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessagesMethodTest.java

```diff
@@ -80,7 +81,42 @@ public void setup() throws Throwable {
     public void teardown() {
         jmapServer.stop();
     }
-    
+
+    @Test
+    public void test() throws Exception {
```

Comment:
Here, I don't especially want to merge it now, but more show you vulnerabilities I found.